packages/merge/src/components/MergeTable/MergeErrors.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import declareFont from 'tocco-ui/src/utilStyles/declareFont'
+
+import sourceDataPropType from '../../util/sourceDataPropType'
+
+const Error = styled.p`
+  && {
+    ${declareFont()}
+    color: #f00;
+  }
+`
+
+const MergeErrors = ({sourceData, mergeErrorMsg, mergeValidationErrors}) => {
+  return <>
+    {
+      !!mergeErrorMsg && <Error>{mergeErrorMsg}</Error>
+    }
+    {
+      mergeValidationErrors.map(e => {
+        const fieldValidators = Object.keys(e.paths)
+          .map(name => Object.keys(e.paths[name]).map(
+            validator => e.paths[name][validator].map((msg, index) =>
+              <Error key={`paths-${name}-${validator}-${index}`}>{sourceData.labels[name]}: {msg}</Error>
+            )
+          ))
+
+        const entityValidators = Object.keys(e.entityValidatorErrors)
+          .map(validator => e.entityValidatorErrors[validator].map((msg, index) =>
+            <Error key={`entity-${validator}-${index}`}>{msg}</Error>
+          ))
+
+        return fieldValidators.concat(entityValidators)
```

Comment:
i prefer

`return [...fieldValidators, ...entityValidators]`

but of course, concat can be used as well